### DISCUSSION TITLE
cannot access argocd-server (insecure mode) (solved)

### DISCUSSION THREAD

**Discussion by kimhongji** [argoproj/argo-cd#7825] [Q&A] [answered]
I deploy argocd-server with --insecure mode like below
```
- command:
        - argocd-server
        - --insecure
```
and I tried to login using argocd cli like ```argocd login <argocd-host>```
but, The rpc error has occured, like
```
WARNING: server is not configured with TLS. Proceed (y/n)? y
Username: admin
Password: 
FATA[0024] rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: <nil> 
```
I have no idea to solve this issue.. there is another way to solve this problem?

---solved---
I just solved that issue by adding `--insecure`, `--grpc-web` option and specified url port.
 -> ```argocd login ‘<host>:80’ --insecure --grpc-web```

**Comment by vikas027** [comment, marked as answer]
Hey @kimhongji , can you please mark this discussion as solved?
